app/src/main/res/layout/fragment_home.xml

```diff
@@ -93,9 +91,7 @@
 
         <Button
             android:id="@+id/tested_button"
-            style="?attr/materialButtonOutlinedStyle"
-            android:layout_width="match_parent"
-            android:layout_height="56dp"
+            style="@style/BigButton"
             android:layout_marginHorizontal="32dp"
```

Comment:
`@dimens/default_horizontal_margin`, `@dimens/default_horizontal_padding` (32dp) everywhere?

Reply:
This is why I labelled this as a "first pass". We have lots of hard-coded dimens in various parts of the app for margins and paddings. I am planning another PR to address positioning dimensions and changing the format of some of our nested `LinearLayout` tags that will include things like gutter margins on screens.

Reply:
(If you feel that this should all be in on PR, I can add to this, however I'm trying to avoid massive PRs by default.)

Reply:
Because you've touched almost all fragment xmls so we can do it in this PR

Reply:
I'll add to the PR as long as no one objects to it being larger.